internal/pkg/template/templates/environment/partials/lambdas.yml

```diff
@@ -12,28 +13,29 @@ CertificateValidationFunction:
     Role: !GetAtt 'CustomResourceRole.Arn'
     Runtime: nodejs12.x
 
-DNSDelegationFunction:
+CustomDomainFunction:
+  Condition: HasAliases
   Type: AWS::Lambda::Function
-  Condition: DelegateDNS
   Properties:
     Code:
       S3Bucket: {{.ScriptBucketName}}
-      S3Key: {{.DNSDelegationLambda}}
-    Handler: "index.domainDelegationHandler"
+      S3Key: {{.CustomDomainLambda}}
+    Handler: "index.handler"
     Timeout: 600
     MemorySize: 512
     Role: !GetAtt 'CustomResourceRole.Arn'
-    Runtime: nodejs12.x
+    Runtime: nodejs12.x 
+{{- end}}
 
-CustomDomainFunction:
-  Condition: HasAliases
+DNSDelegationFunction:
```

Comment:
Why do we want the DNSDelegation Lambda still created, wouldn't the customer be in charge of all the hosted zones and the records when they import a cert?

Reply:
Discussed offline. Will keep it if they have domain specified.

Reply:
> Discussed offline. Will keep it if they have domain specified.

Seems like it's not kept when import cert happens! This makes sense to me per our offline discussion but I just want to make it clear here in case others are reviewing!